_posts/2020/2020-02-02-calculate-a-mean-average-from-a-ruby-array.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Calculate a mean average from a Ruby array"
+description: "A lesson in why we use native Ruby methods"
+layout: article
+category: ruby
+image:
+  base: '2020/calculate-a-mean-average-from-a-ruby-array'
+  alt: "Black and White Texas Instruments calculator"
+  credit:  Ray Reyes
+  source: "https://unsplash.com/photos/3xwrg7Vv6Ts"
+
+---
+
+Ruby doesn’t provide a native method to generate an average (mean) value from an array of integers. Its built-in [math](https://ruby-doc.org/core-2.7.0/Math.html) library deals with more complex calculations than that and there's no native `#average` or `#mean` method on `Array`.
+
+This leaves us free to create our own implementation, with opportunities to shoot ourselves in the foot.
```

Comment:
```suggestion
This leaves us free to create our own implementation, bringing with that opportunities to shoot ourselves in the foot.
```

Not convinced on either mine or your version of this sentence...

Reply:
```suggestion
This leaves us free to create our own implementation, bringing the opportunity to shoot ourselves in the foot.
```